README.md

```diff
@@ -534,6 +534,9 @@ Name | Type | Description
 action | string | the action that occurred (add, delete, modify)
 annotations | array | array of annotation data in the format `{id: string, pageNumber: number}`
 
+##### annotationPermissionCheckEnabled
+bool, optional, default to false If true, annotation created by user A cannot be modified by user B, else anyone can modify any annotation.
```

Comment:
if true should go new line, you can click on your branch on github to check how it looks

Reply:
change to 

If true, annotation's flags will be taken into account when it is selected, for example, a locked annotation can not be resized or moved.

Reply:
yes fix all, if true should always be on new line